iop/system/udnl-t300/src/IOPRP_img.c

```diff
@@ -0,0 +1,8 @@
+#ifndef __IOPRP_img__
+#define __IOPRP_img__
+
+unsigned int size_IOPRP_img = 0;
+unsigned char IOPRP_img[] __attribute__((aligned(16))) = {
+};
+
+#endif
```

Comment:
In OPL this file is generated from IOPRP.img using extra makefile rules?

Reply:
The functionality was made optional, so this file was removed.